staging/prometheus-operator/.helmignore

```diff
@@ -24,3 +24,5 @@ OWNERS
 hack/
 ci/
 prometheus-operator-*.tgz
+# Mesosphere-specific files to ignore
+patch/
```

Comment:
Whats the function of the `staging/prometheus-operator/patch/patch_10_mesosphere_helmignore.sh` file, is this not enough?

Reply:
Since this is our fork of the upstream chart, we want to include any modifications we make on top of the upstream chart in these patch files to make upgrading easier - so in the next upgrade, we can just directly apply these patches on top of the updated chart (like a rebase) instead of manually going through and re-applying each change.